test/TestApps/AppLifecycleTestApp/Helpers.cpp

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include "pch.h"
+#include "Helpers.h"
```

Comment:
>Helpers [](http://example.com/codeflow?start=10&length=7)

Not fond of the name :-(

Personal nit. Helper, Util and Temp are usually too blandly generic and makes me sad.